src/main/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/v2/DataProvider.kt

```diff
@@ -0,0 +1,18 @@
+package it.pagopa.ecommerce.helpdesk.dataproviders.v2
```

Comment:
Why duplicate this interface? this can be leaved outside versioning and make both V1 and V2 `TransactionDataProvider` interfaces extend the common `DataProvider`

Reply:
done here  [commit](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/118/commits/6a6778379c4172008f1a249de8bf057d50899f70)